pkg/datasource/query_handler.go

```diff
@@ -342,18 +343,23 @@ func fillQueryVariables(query string, databaseType string, backendQuery backend.
 	return query
 }
 
-func historianQuery(query schemas.MeasurementQuery, backendQuery backend.DataQuery) schemas.Query {
+func historianQuery(query schemas.MeasurementQuery, historianInfo *schemas.HistorianInfo, backendQuery backend.DataQuery) schemas.Query {
 	start := backendQuery.TimeRange.From.Truncate(time.Second)
 	end := backendQuery.TimeRange.To.Truncate(time.Second)
 	historianQuery := schemas.Query{
 		MeasurementUUIDs: query.Measurements,
 		Start:            start,
 		End:              &end,
 		Tags:             query.Options.Tags,
+		ValueFilters:     query.Options.ValueFilters,
 		GroupBy:          query.Options.GroupBy,
 		Format:           schemas.ArrowFormat,
 	}
 
+	if historianInfo != nil && util.SemverCompare(historianInfo.Version, "7.1.0") {
+		historianQuery.ValueFilters = query.Options.ValueFilters
+	}
+
```

Comment:
remove, check is already done in the frontend and 5 lines above you set it anyway